Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -203,6 +204,83 @@
             },
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "version": "KqlParameterItem/1.0",
+            "name": "subscriberLimit",
+            "type": 1,
+            "query": "resources\r\n| where id =~ \"{pccp:id}\"\r\n| extend sku=tostring(properties.sku)\r\n| extend skuMaxSubscribers = iff(sku==\"G0\", \"20\", iff(sku==\"G1\", \"100\", iff(sku==\"G2\", \"200\", iff(sku==\"G5\", \"500\", iff(sku==\"G10\", \"1000\", \"Unknown\")))))\r\n| project [\"Subscriber Limit\"]=skuMaxSubscribers",
```

Comment:
It would be good for maintainability to reduce the amount of duplication of the subscriber/nodeb limits. Can you just make the queries once and then refer to those values when calculating the other parameters?